test/Dialect/ESI/services.mlir

```diff
@@ -17,10 +18,75 @@ hw.module @Top (%clk: i1) -> (chksum: i8) {
 // CHECK:         %0 = esi.service.instance @HostComms impl as "topComms"(%clk) : (i1) -> i8
 // CHECK:         hw.instance "m1" @Loopback(clk: %clk: i1) -> ()
 
+// CONN-LABEL: hw.module @Top(%clk: i1) -> (chksum: i8) {
```

Comment:
Nit: most tests in CIRCT/MLIR follow a style of checks-before-code (I think i've only come across code-before-checks in the MSFT/ESI tests).